error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1204,12 +1204,12 @@ StepVerifier.FirstStep<? extends T> after(Flux<T> flux) {
     }
   }
 
-  /** Don't unnecessarily call {@link StepVerifier.Step#expectNext(Object[])}. */
-  static final class StepVerifierStepExpectNextEmpty<T> {
+  /** Don't unnecessarily have {@link StepVerifier.Step} expect no elements. */
+  static final class StepVerifierStepIdentity<T> {
     @BeforeTemplate
     @SuppressWarnings("unchecked")
     StepVerifier.Step<T> before(StepVerifier.Step<T> step) {
-      return step.expectNext();
+      return Refaster.anyOf(step.expectNext(), step.expectNextCount(0L));
```

Comment:
I see that both `0` and `0L` cover both `int`s and `long`s. We can use the `int` here, and then test against the `long`.
```suggestion
      return Refaster.anyOf(step.expectNext(), step.expectNextCount(0));
```